edb/lang/edgeql/ast.py

```diff
@@ -127,6 +127,14 @@ class Cardinality(s_enum.StrEnum):
     DEFAULT = ''
 
 
+class Cascade(s_enum.StrEnum):
```

Comment:
I think `LinkTargetDeleteAction` is a more appropriate name for this enumeration.